Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoOutput.kt

```diff
@@ -336,3 +336,5 @@ data class CertificatesRevokedByInvalidationRules(
     @JsonProperty("certificateRef")
     val certificateRef: CertificateRef
 )
+
+internal const val DEFAULT_IDENTIFIER = "renew"
```

Comment:
We create actions in several places. Maybe we should create an Enum for the actions.